eng/common/TestResources/New-TestResources.ps1

```diff
@@ -143,6 +143,14 @@ function Retry([scriptblock] $Action, [int] $Attempts = 5)
 # https://azure.microsoft.com/en-us/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/
 function NewServicePrincipalWrapper([string]$subscription, [string]$resourceGroup, [string]$displayName)
 {
+    if ((Get-Module Az.Resources).Version -eq "5.3.0") {
```

Comment:
Should we consider updating the global requirement on L9 rather than special casing, or would that cause too much churn to existing callers?

Reply:
We could, but so far all the work has been done to support other 4.x and 5.x versions, so I don't see a pressing need to force people to update (other than the pending AAD graph deprecation). As there has been churn on basically every single new version of Az.Resources + MS Graph related cmdlets, I think if we do force an upgrade, we should wait until they settle on something stable, otherwise we'll have to keep adding these special cases or asking people to upgrade.

Reply:
Yeah.... that's fair.